addons/fullscreen-editor/addon.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Fullscreen editor",
+  "description": "Adds a fullscreen toggle to the editor, and improves the layout when in fullscreen mode.",
+  "tags": ["editor"],
+  "versionAdded": "1.31.0",
+  "userscripts": [
+    {
+      "matches": ["projects"],
+      "runAtComplete": true,
+      "url": "userscript.js"
+    }
+  ],
+  "userstyles": [
+    {
+      "matches": ["projects"],
+      "url": "userstyle.css"
+    }
+  ],
+  "settings": [
+    {
+      "id": "shrinkStage",
```

Comment:
Could this setting be hidden when the hide stage setting is enabled?

Reply:
Not with the way that the settings page is currently set up, I'll work on fixing that.

Reply:
`"if": {"settings": {"hide": false }}`?